tests/tracing/test_pruning.py

```diff
@@ -1242,7 +1241,7 @@ def test_prune_by_length(
         pytest.param(
             "pruning_skeleton",
             None,
-            8.0e-19,
+            7.3,
```

Comment:
Why has the value as well as the magnitude changed for this set of parameters?

Reply:
As seen above, some of the values changed slightly so I modified the test values in order to keep the expected output the same. When doing this I was logging the height values produced by the `method_values` to find one that fits but for the value of ~8e-19, the largest branch height was ~7.9 so couldn't be used.